ci/azure_install_macos.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 conda config --set always_yes yes --set changeps1 no
 conda info -a
 conda update -q conda
-conda install -c conda-forge python=3.8 re2c bison=3.4 m4 xonsh llvmdev=11.0.1 toml cmake=3.17.0 jupyter pytest xeus=1.0.1 xtl nlohmann_json cppzmq jupyter_kernel_test
+conda install -c conda-forge python=3.8 re2c bison=3.4 m4 xonsh llvmdev=11.0.1 toml cmake=3.17.0 jupyter pytest xeus=1.0.1 xtl nlohmann_json cppzmq jupyter_kernel_test symengine sympy
```

Comment:
I would install specific versions of sympy and symengine.

Reply:
I think we don't use this and so it would be good to remove this too.